dll/cpl/usrmgr/users.c

```diff
@@ -672,6 +717,41 @@ UsersPageProc(HWND hwndDlg,
             pUserData->hPopupMenu = LoadMenu(hApplet, MAKEINTRESOURCE(IDM_POPUP_USER));
 
             OnInitDialog(hwndDlg);
+			
+            HKEY hKeyInit;
```

Comment:
Please put the `case WM_INITDIALOG:` code block inside brackets:
```c
case WM_INITDIALOG:
{
    // The code ...
    break;
}

case WM_COMMAND:
...
```
and the variable declarations at the top of the block.